Dockerfiles/midstream/Dockerfile

```diff
@@ -1,20 +1,17 @@
 FROM registry.fedoraproject.org/fedora:33
-WORKDIR /project
 ARG OPERATOR_SDK_VERSION=v1.4.0
-ARG OPERATOR_TEST_PLAYBOOKS_TAG=v1.0.11
+ARG OPERATOR_TEST_PLAYBOOKS_TAG=master
 ARG UMOCI_VERSION=v0.4.5
-ADD ./fix_etc_passwd.sh /usr/bin/fix_etc_passwd.sh
 RUN export ARCH=$(case $(arch) in x86_64) echo -n amd64 ;; aarch64) echo -n arm64 ;; *) echo -n $(arch) ;; esac);\
     export OS=$(uname | awk '{print tolower($0)}');\
     export OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/download/$OPERATOR_SDK_VERSION/;\
     curl -L -o /usr/local/bin/operator-sdk ${OPERATOR_SDK_DL_URL}/operator-sdk_${OS}_${ARCH} && \
     chmod a+x /usr/local/bin/operator-sdk && \
     curl -fL -o /usr/local/bin/umoci https://github.com/opencontainers/umoci/releases/download/${UMOCI_VERSION}/umoci.amd64 && \
     chmod a+x /usr/local/bin/umoci && \
-    mkdir /project/output && \
+    mkdir /tmp/output && \
     chmod g+w /etc/passwd && \
     dnf install --setopt=install_weak_deps=False -y git-core ansible skopeo && \
-    dnf clean all
-ADD ./run_tests.py /run_tests.py
-ADD ./unit_tests.py /unit_tests.py
-CMD ["/run_tests.py"]
+    dnf clean all 
+ADD . /tmp/operator-test-playbooks/
```

Comment:
I not not like such approach. Files shoudl be sotred in righ directories an not in top level (`/`)
e.g.
```
+ADD ./Dockerfiles/midstream/run_tests.py ./Dockerfiles/midstream/unit_tests.py ./Dockerfiles/midstream/fix_etc_passwd.sh /usr//bin/
+ADD roles /project/operator-test-playbooks/roles/
+ADD extract-operator-bundle.yml validate-operator-bundle.yml /project/operator-test-playbooks/
+CMD ["/usr/bin/run_tests.py"]

```

Reply:
Yes I agree. However, In order to bypass the permission issue we are currently facing right now This approach was working.

Reply:
If plabooks and roles are stored in `/project/` or in `/usr/share/ansible` ... it does not mean you need to
write temporary files in `project` or in `/usr/share/`

You should still write temporary files either to CWD (or homedir) or rirectly into /tmp or /var/tmp

Reply:
this is still not solved

Reply:
The files are now not written in temporary directory.